scripts/changeset/changeset-version-with-docs.ts

```diff
@@ -16,8 +16,13 @@ import { error } from 'console';
      */
     execSync(`changeset version`);
 
-    // Invoke versions' prebuild script (will rewrite version files if needed)
-    execSync(`pnpm -C packages/versions prebuild`);
+    /**
+     * Invoke versions' build script (will rewrite version files if needed)
+     * and build the versions package,
+     * so that fuels-typegen picks up the new fuels version
+     * when generating the proxy contract below.
```

Comment:
Just equalizing the line breaks.

```suggestion
     * Invoke versions' build script (will rewrite version files if needed)
     * and build the versions package, so that fuels-typegen picks up the
     * new fuels version when generating the proxy contract below.
```